src/main/java/seedu/address/logic/commands/DeleteLogCommand.java

```diff
@@ -188,19 +214,45 @@ public CommandResult deleteAllLogs(Model model) {
          */
         public CommandResult deleteAllLogsOfPerson(Model model) throws CommandException {
 
-            // sanity check
-            assert (this.isForDeletingAllLogs && this.isForOnePerson
-                    && this.personIndex != null && this.logIndex == null);
+            // sanity checks
+            assert (this.isForDeletingAllLogs && this.isForOnePerson && this.logIndex == null);
+            assert (this.personIndex != null || this.personWithNameToDeleteLog != null);
+
+            // ===== GET PERSON =====
```

Comment:
Looks like this is duplicated - is it possible to abstract this out into a method?